tutorials/buoyantSimpleFoam-aster/precice-config.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<precice-configuration>
+  <log>
+     <sink filter="%Severity% > debug" format="---[precice] %ColorizedSeverity% %Message%" enabled="true"/>
+  </log>
+  <solver-interface dimensions="3">
+    <data:scalar name="Heat-Transfer-Coefficient-Solid"/>
+    <data:scalar name="Sink-Temperature-Solid"/>
+    <data:scalar name="Heat-Transfer-Coefficient-Fluid"/>
+    <data:scalar name="Sink-Temperature-Fluid"/>
+    <mesh name="Solid-Interface-Nodes">
+      <use-data name="Sink-Temperature-Solid"/>
+      <use-data name="Heat-Transfer-Coefficient-Solid"/>
+    </mesh>
+    <mesh name="Solid-Interface-Faces">
+      <use-data name="Sink-Temperature-Fluid"/>
+      <use-data name="Heat-Transfer-Coefficient-Fluid"/>
+    </mesh>
+    <mesh name="Fluid-Interface">
+      <use-data name="Sink-Temperature-Fluid"/>
+      <use-data name="Heat-Transfer-Coefficient-Fluid"/>
+      <use-data name="Sink-Temperature-Solid"/>
+      <use-data name="Heat-Transfer-Coefficient-Solid"/>
+    </mesh>
+    <participant name="Solid">
+      <use-mesh name="Solid-Interface-Faces" provide="yes"/>
+      <use-mesh name="Solid-Interface-Nodes" provide="yes"/>
+      <use-mesh name="Fluid-Interface" from="Fluid"/>
+      <write-data mesh="Solid-Interface-Nodes" name="Sink-Temperature-Solid"/>
+      <write-data mesh="Solid-Interface-Nodes" name="Heat-Transfer-Coefficient-Solid"/>
+      <read-data mesh="Solid-Interface-Faces" name="Sink-Temperature-Fluid"/>
+      <read-data mesh="Solid-Interface-Faces" name="Heat-Transfer-Coefficient-Fluid"/>
+      <mapping:nearest-neighbor constraint="consistent" direction="read" to="Solid-Interface-Faces" from="Fluid-Interface"/>
+    </participant>
+    <participant name="Fluid">
+      <use-mesh name="Fluid-Interface" provide="yes"/>
+      <use-mesh name="Solid-Interface-Nodes" from="Solid"/>
+      <write-data mesh="Fluid-Interface" name="Sink-Temperature-Fluid"/>
+      <write-data mesh="Fluid-Interface" name="Heat-Transfer-Coefficient-Fluid"/>
+      <read-data mesh="Fluid-Interface" name="Sink-Temperature-Solid"/>
+      <read-data mesh="Fluid-Interface" name="Heat-Transfer-Coefficient-Solid"/>
+      <mapping:nearest-neighbor constraint="consistent" direction="read" to="Fluid-Interface" from="Solid-Interface-Nodes"/>
+    </participant>
+    <m2n:sockets to="Fluid" from="Solid" exchange-directory="/home/michel/Desktop/Code_Aster-Adapter-Dummy/buoyantSimpleFoam-aster" distribution-type="gather-scatter"/>
```

Comment:
We don't / should not need this hard-coded path. It defaults to the current directory.

```suggestion
    <m2n:sockets to="Fluid" from="Solid" distribution-type="gather-scatter"/>
```

Reply:
Both adapters are unable to setup a master communication to the coupling partner when this change is implemented.